scripts/irods/test/test_icommands_file_operations.py

```diff
@@ -64,6 +64,25 @@ def test_re_serialization__prep_13(self):
         finally:
             IrodsController().restart()
 
+    @unittest.skipUnless(plugin_name == 'irods_rule_engine_plugin-python', 'only applicable for python REP')
+    def test_re_serialization__prep_55(self):
+        try:
+            IrodsController().stop()
+            initial_size_of_server_log = lib.get_file_size_by_path(paths.server_log_path())
+            with temporary_core_file() as core:
+                core.add_rule(rule_texts[self.plugin_name][self.class_name][inspect.currentframe().f_code.co_name])
+                IrodsController().start()
+                with tempfile.NamedTemporaryFile(prefix='test_re_serialization__prep_13') as f:
```

Comment:
test_re_serialization__prep_13 should be 55?

Reply:
Done! And the corresponding one in 4-2-stable